File: ApiSuvesaPos/Negocio/Modelo/inventario/cambioinventario.vb
Namespace Modelo
    Public Class cambioinventario
        Public Property Id As Integer
        Public Property IdUsuario As String
        Public Property Fecha As DateTime
        Public Property CodigoFacturado As Integer
        Public Property CantidadFacturado As Decimal
        Public Property CodigoEntregado As Integer
        Public Property CantidadEntregado As Decimal
        Public Property Diferencia As Decimal
        Public Property Aplicado As Boolean
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/cambioinventario.cs
using System;

namespace Modelo
{
    public class cambioinventario
    {
        public int Id { get; set; }
        public string IdUsuario { get; set; }
        public DateTime Fecha { get; set; }
        public int CodigoFacturado { get; set; }
        public decimal CantidadFacturado { get; set; }
        public int CodigoEntregado { get; set; }
        public decimal CantidadEntregado { get; set; }
        public decimal Diferencia { get; set; }
        public bool Aplicado { get; set; }
    }
}
